_assets/js/carousel.js

```diff
@@ -0,0 +1,335 @@
+/*!
+ * Adapted from W3C WAI’s Web Accessibility Tutorials carousel
+ * https://github.com/w3c/wai-tutorials/
+ */
+
+const mobileCarousel = () => {
+  // Initial constiables
+  let carousel;
+  let slides;
+  let index;
+  let slidenav;
+  let settings;
+  let timer;
+  let setFocus;
+  let animationSuspended;
```

Comment:
if it's necessary to have a global context like this, consider namespacing it so it's more obvious where the variables come from when we're 200 lines down the page.

```suggestion
  const context = {
      carousel: undefined,
      slides: undefined,
      index: undefined,
      slidenav: undefined,
      settings: undefined,
      timer: undefined,
      setFocus: undefined,
      animationSuspended: undefined,
  };
```

Also consider giving them sensible defaults (animationSuspended probably makes more sense as `false` than `undefined`